src/main/java/com/app/service/graph/GraphService.java

```diff
@@ -1,32 +1,140 @@
 package com.app.service.graph;
 
+import javafx.fxml.FXML;
+import javafx.scene.Parent;
 import javafx.scene.layout.AnchorPane;
+import javafx.stage.FileChooser;
 import javafx.stage.Stage;
+import org.jfree.chart.JFreeChart;
 import org.jfree.chart.fx.ChartViewer;
 
+import java.awt.*;
+import java.io.File;
+import java.io.FileNotFoundException;
+
 
 public class GraphService {
-    AnchorPane anchorPane;
-    Stage stage;
-    int poc = 0;
+    AnchorPane anchorPaneUpper;
+    AnchorPane anchorPaneLower;
+    Button upperLoadButton;
+    Button lowerLoadButton;
+    private STATE stateUpper = STATE.NOTRUNNING;
+    private STATE stateLower = STATE.NOTRUNNING;
+    boolean running = false;
+    String loadingTo = null;
+    ChartViewer chartViewerUpper = new ChartViewer();
+    ChartViewer chartViewerLower = new ChartViewer();
+
 
-    public GraphService(AnchorPane aP, Stage s) {
-        anchorPane = aP;
-        stage = s;
+    public enum STATE {
```

Comment:
enum nie je dobre zabalovat do triedy, radsej to vytiahni von, ako sme to robili aj vsade inde, taktiez je prehladnesie ak pomenujes UPPER_RUNNING namiesto UPPERRUNNING, vsak? Su to detailiky, ale ulahci a zrychli to pracu aj pre tych, co citaju po tebe :)